code-challenges.test.js

```diff
@@ -49,4 +145,49 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+describe("fullHouse", () => {
+    it("function that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.", () => {
+      expect(fullHouse(hand1)).toEqual(true)
+      expect(fullHouse(hand2)).toEqual(false)
+      expect(fullHouse(hand3)).toEqual(false)
+      expect(fullHouse(hand4)).toEqual(true)
+    })
+  })
+
+  //    ReferenceError: fullHouse is not defined
+
+
+
 // b) Create the function that makes the test pass.
+
+/* This first method of using for look and conditional at least in this first way was only party effective. only worked for the first and last array. 
+*/
+
+// const fullHouse = (array) => {
+//     for (i=0; i < array.length; i++)
+//     if (array.indexOf(array[i], 3)){
+//     return true
+//     }
+//     else (array.indexOf(array[i], 4))
+//     { 
+//     return false
+//     }
+//  }
+
+/*
+This took a while, to find something that works. I found this fullHouse code that is pretty cool! I dind't know you could do this, or maybe it just didn't click until now, but you cn use an object as something to memorize things. So as you iterate through the for loop. the + 1 is for it to keep going. I think this is really cool and useful for future situations for sure!
```

Comment:
The plus one might be misleading.  In this case, the plus 1 is not to advance to the next iteration, it is to update the value in the countObj object.  I would really like to see a breakdown of your code line by line for your psuedocode.